src/main/java/shop/menu/CredentialsMenu.java

```diff
@@ -23,6 +23,11 @@ public String getEmail() {
         return email;
     }
 
+    public String getId(){
```

Comment:
Id of what? Too abstract. I assume it’s id for order? Why in CredentialsMenu?) how is it related?